tests/backup/backup_delete_test.go

```diff
@@ -15,6 +16,85 @@ import (
 	. "github.com/portworx/torpedo/tests"
 )
 
+// createBackupUntilIncrementalBackup creates backup until incremental backups is created
+func createBackupUntilIncrementalBackup(namespace string, customBackupLocationName string, backupLocationUID string, labelSelectors map[string]string, orgID string, clusterUid string, ctx context.Context) string {
+	incrementalBackupName := fmt.Sprintf("%s-%s-%v", "incremental-backup", namespace, time.Now().Unix())
```

Comment:
since you are returning string, add error also and return error.
 Also is it spec to particular test, if not you can move to backup/common

Reply:
Done @lsrinivas-pure